python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/logging.j2

```diff
@@ -39,6 +27,18 @@ logging event storm-control discards global
 {%     if logging.event.storm_control.discards.interval is arista.avd.defined %}
 logging event storm-control discards interval {{ logging.event.storm_control.discards.interval }}
```

Comment:
`logging event storm-control discards` commands not come along with others. please check

Reply:
Moved this to a new template.